src/site/content/en/progressive-web-apps/define-install-strategy/index.md

```diff
@@ -0,0 +1,169 @@
+
+You can achieve this in different ways:
+
+- Installing the PWA [from the browser](https://web.dev/customize-install/).
+- Installing the PWA [from the app store](https://developers.google.com/web/android/trusted-web-activity).
+
+Having different distribution channels is a powerful way of reaching a broad number of users, but choosing the right strategy to promote them, can be challenging.
+
+This guide explores best practices for combining different installation offerings to increase installation rates and avoid platform competition and cannibalization. The installation offerings covered include PWAs installed from both the browser and the App Store, as well as native apps.
+
+## Why make your web app installable?
+
+Installed Progressive Web Apps run in a standalone window instead of a browser tab. They're launchable from on the user's home screen, dock, taskbar, or shelf. It's possible to search for them on a device and jump between them with the app switcher, making them feel like part of the device they're installed on. 
+
+But having both an installable web app, and a native app can be confusing for users. For some users native apps may be the best choice, but for others they can present some drawbacks:
+
+- **Storage constraints:** Installing a new app might mean deleting others, or cleaning up space, by removing valuable content. This is especially disadvantageous for users on low-end devices.
+- **Available bandwidth:** Downloading an app can be a costly and slow process, even more for users on slow connections and expensive data plans.
+- **Friction:** Leaving a website and moving to a store to download an app, creates additional friction, delaying a user action that could be performed directly in the web.
+- **Update cycle:** Making changes in native apps might require going through an app review process, which can slow down changes and experiments (e.g. A/B tests).
+
+In some cases, the percentage of users that won't download your native app might be large, for example: those that think that they won't use the app very often, or can't justify spending several megabytes of storage or data. You can determine the size of this segment in several ways, for example by using an analytics setup to track the percentage of "mobile web only" users.
+
+If the size of this segment is considerable, that's a good indication that you need to provide alternative ways of installing your experiences.
+
+## Promoting the installing of your PWA through the browser
+
+If you have a high quality PWA, it may be better to promote its installation over your native app. For example, if the native app is missing functionality offered by your PWA, or if it hasn't been updated in a while. It can also be helpful to promote installation of your PWA if the native app wasn't optimized for bigger screens, such as on ChromeOS.
```

Comment:
```suggestion
If you have a high quality PWA, it may be better to promote its installation over your native app. For example, if the native app is missing functionality offered by your PWA, or if it hasn't been updated in a while. It can also be helpful to promote installation of your PWA if the native app wasn't optimized for bigger screens, such as on Chrome OS.
```